rcav2/api.py

```diff
@@ -86,6 +86,10 @@ async def run(self, worker: Worker):
         rcav2.database.set_job(self.db, self.name, json.dumps(list(events)))
 
 
+router = APIRouter()
+
+
+@router.put("/get_job")
```

Comment:
This is counter-intuitive. If it's really a function which requires `put` I would recommend the change:
```
@router.put("/job")
```
However this function doesn't seem to be doing anything `put` related: 
> The PUT HTTP method creates a new resource or replaces a representation of the target resource with the request [content](https://developer.mozilla.org/en-US/docs/Glossary/HTTP_Content).

[src](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/PUT)